src/assets/js/protocol/protocol-notification-bar.js

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mzp === 'undefined') { // eslint-disable-line block-scoped-var
+    var Mzp = {};
+}
+
+Mzp.Notification = (function() { // eslint-disable-line block-scoped-var
+    'use strict';
+
+    var options = {};
+
+    /*
+    origin: element that triggered the notification.
+    options: object of params:
+        title: title to display inside the notification. [required]
+        className: CSS class name to apply to the notification.
+        closeText: 'string to use for close button a11y.
+        hasDismiss: boolean - include or not include dismiss button.
+        isSticky: boolean - determines if notification is absolutely positioned and sticky.
+        onNotificationOpen: function to fire after notification has been opened.
+        onNotificationClose: function to fire after notification has been closed.
+    */
+
+    var _init = function(origin, opts) {
+
+        if (typeof opts === 'object') {
+            for (var i in opts) {
+                if (opts.hasOwnProperty(i)) {
+                    options[i] = opts[i];
+                }
+            }
+        }
+
+        // Create new notification
+        var title = document.createTextNode(options.title);
+        var className = (options && options.className) ? options.className : '';
+        var closeText = (options && options.closeText) ? options.closeText : '';
+        var isSticky = (options && options.isSticky) ? 'mzp-is-sticky' : '';
+
+        var notification = document.createElement('aside');
+        notification.className = 'mzp-c-notification-bar ' + className + ' ' + isSticky;
+
+        // Notification Title
+        if (options && options.title){
+            var notificationTitle = document.createElement('p');
+            notificationTitle.className = 'mzp-c-notification-bar-content';
+            notificationTitle.appendChild(title);
+
+            // add title to notification
+            notification.appendChild(notificationTitle);
+        }else{
+            console.log('Notification banner requires title parameter.'); // eslint-disable-line no-console
```

Comment:
I don't think we should leave `console.log()` in as production code, since it can cause errors in older browsers unless dev tools are open.

Reply:
agreed! will fix that up tomorrow.